hips/draw/tests/test_simple.py

```diff
@@ -53,5 +55,34 @@ def test_make_sky_image():
     assert_allclose(data[200, 994], 3717.10091363)
     assert_allclose(data[200, 995], 3402.55292158)
 
+get_hips_order_pars = [
```

Comment:
To me, `get_hips_order_pars` sounds like a function. This is just a list, so suggest to rename to `hips_order_pars`.